data/s-prateek.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Prateek Sharma",
+  "type": "personal",
+  "bio": "Lead Full Stack Engineer @3M | AWS | Azure | Blockchain | Python | .Net | React",
+  "links": [
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/-prateek-sharma/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on GitHUb",
+      "url": "https://github.com/s-prateek",
+      "icon": "FaGit"
```

Comment:
Suggestion: use `FaGithub` icon than `FaGit` 
Check out more icons here https://linkfree.io/icons

Reply:
As this is not a blocker I will merge this Pull Request. 

@s-prateek should you wish to make changes to your profile in future, please raise a new Pull Request.  You may wish to take a look at the Editing Guide in Docs: https://linkfree.io/docs/how-to-guides/editing